frontend/public/components/utils/details-page.tsx

```diff
@@ -23,7 +23,7 @@ export const ResourceSummary: React.SFC<ResourceSummaryProps> = ({children, reso
     { type ? <dt>Type</dt> : null }
     { type ? <dd>{type}</dd> : null }
     <dt>Labels</dt>
-    <dd><LabelList kind={referenceFor(resource)} labels={metadata.labels} /></dd>
+    <dd><LabelList kind={resource.kind} labels={metadata.labels} /></dd>
```

Comment:
nit: We could add this to the destructing assignment above.

It would be better if the search page handled the fully-qualified resource strings, though. `kind` is not unique.

Reply:
Agreed, I was planning on [fixing the search page soon](https://jira.coreos.com/browse/CONSOLE-678). May need to accelerate that effort.